content/guides/upgrading-to-tf12-tg19/index.md

```diff
@@ -0,0 +1,340 @@
+   page](https://github.com/gruntwork-io/terratest/releases).
+1. Push your changes to a branch so that you can test it in your pre-prod environments (We will come back to this later
+   in the guide, when we upgrade the live config to TG19).
+
+
+### Upgrade live config for compatibility with Terragrunt 0.19.x
+
+Terragrunt 0.19.x introduces a new file format for the live config. This format has many advantages, including
+supporting all built in functions available to Terraform. However, this means that you will need to convert all your
+terragrunt `terraform.tfvars` files to the new syntax.
+
+Here is the rough process:
+
+1. [Install and setup Terragrunt 0.19](#install-and-setup-terragrunt-0-19)
+1. [Migrate terraform.tfvars to terragrunt.hcl](#migrate-terraform-tfvars-to-terragrunt-hcl)
+1. [Switch common tfvars files to use yaml](#switch-common-tfvars-files-to-use-yaml)
+
+#### Install and setup Terragrunt 0.19
+
+You will also need to setup the Terragrunt 0.19 binary in a [similar fashion to setting up Terraform
+0.12](#install-and-setup-terraform-0-12). You can use the same instructions available for managing Terraform 0.12 as for
+managing Terragrunt 0.19, except for `tfenv` which only supports Terraform
+
+- [Using homebrew to manage multiple terragrunt versions](#using-homebrew-to-manage-multiple-terraform-versions)
+- [Manually managing multiple terragrunt versions](#manually-managing-multiple-terragrunt-versions)
+
+##### Using homebrew to manage multiple terragrunt versions
+
+Like `terraform`, you can use Homebrew to manage multiple `terraform` versions, using the same method described above in
+[Using homebrew to manage multiple terraform versions](#using-homebrew-to-manage-multiple-terraform-versions). To use
+the method for terragrunt, replace the references for `terraform` with `terragrunt`. For example, in the first step when
+searching for the homebrew commit that introduces the Terragrunt version for `0.18.7`, you would run:
+
+```
+$ git log master -- Formula/terragrunt.rb
+```
+
+instead of the equivalent one for Terraform.
+
+##### Manually managing multiple terragrunt versions
+
+Like `terraform`, you can use the method described above in [Manually managing multiple terraform
+versions](#manually-managing-multiple-terraform-versions) to manage multiple versions of Terragrunt.
+
+You can find the available binaries for each terragrunt version
+[here](https://github.com/gruntwork-io/terragrunt/releases).
+
+
+#### Migrate terraform.tfvars to terragrunt.hcl
+
+Once you have terragrunt 0.19 available on your machine, you will need to migrate your `terraform.tfvars` files to
+`terragrunt.hcl` files before you can start to use the new version. You will need to follow the steps outlined in the
+[Terragrunt 0.19.x migration
+guide](https://github.com/gruntwork-io/terragrunt/blob/master/_docs/migration_guides/upgrading_to_terragrunt_0.19.x.md)
+for each of your live config (e.g in your `infrastructure-live` repo).
```

Comment:
GitHub's diff view shows all of the text from here onward in italics. GitHub bug or extra `_` somewhere?

Reply:
Looking at the rendering, it's rendering italics based on the underscores in the URL link above :(